api/tests/integration/infrastructure/repositories/organization-repository_test.js

```diff
@@ -1,41 +1,134 @@
-const { expect, knex, sinon } = require('../../../test-helper');
+const { expect, knex, sinon, factory, databaseBuilder } = require('../../../test-helper');
 const faker = require('faker');
 const bcrypt = require('bcrypt');
 const Organization = require('../../../../lib/domain/models/Organization');
+const BookshelfOrganization = require('../../../../lib/infrastructure/data/organization');
 const organizationRepository = require('../../../../lib/infrastructure/repositories/organization-repository');
 
 describe('Integration | Repository | Organization', function() {
 
   describe('#create', () => {
 
-    const userPassword = bcrypt.hashSync('A124B2C3#!', 1);
-    const inserted_user = {
-      firstName: faker.name.firstName(),
-      lastName: faker.name.lastName(),
-      email: faker.internet.email(),
-      password: userPassword,
-      cgu: true
-    };
+    afterEach(() => {
+      return knex('organizations').delete();
+    });
 
-    beforeEach(() => {
-      return knex('users').insert(inserted_user);
+    it('should return an Organization domain object', () => {
+      // given
+      const organization = factory.buildOrganization();
+
+      // when
+      const promise = organizationRepository.create(organization);
+
+      // then
+      return promise.then((organizationSaved) => {
+        expect(organizationSaved).to.be.an.instanceof(Organization);
+      });
     });
 
-    afterEach(() => {
-      return knex('users').delete();
+    it('should add a row in the table "organizations"', async () => {
+      // given
+      const nbOrganizationsBeforeCreation = await BookshelfOrganization.count();
+
+      // when
+      const promise = organizationRepository.create(factory.buildOrganization());
+
+      // then
+      return promise.then(async () => {
+        const nbOrganizationsAfterCreation = await BookshelfOrganization.count();
+        expect(nbOrganizationsAfterCreation).to.equal(nbOrganizationsBeforeCreation + 1);
+      });
     });
 
-    it('should save model in database', () => {
+    it('should save model properties', async () => {
       // given
-      const organization = new Organization({ code: 'AAAA99', name: 'Lycée Rousseau', type: 'SCO' });
+      const organization = factory.buildOrganization({ id: null });
 
       // when
       const promise = organizationRepository.create(organization);
 
+      // then
+      return promise.then(async (organizationSaved) => {
+        expect(organizationSaved.id).to.not.be.undefined;
```

Comment:
Il y a un return ici, mais un `async` en haut dans le `it`. Est-ce que c'est plutôt un `await` ?